codecov_auth/commands/owner/interactors/delete_session.py

```diff
@@ -7,14 +7,24 @@
 
 
 class DeleteSessionInteractor(BaseInteractor):
-    def validate(self):
+    def validate(self) -> None:
         if not self.current_user.is_authenticated:
             raise Unauthenticated()
 
     @sync_to_async
-    def execute(self, sessionid: int):
+    def execute(self, sessionid: int) -> None:
```

Comment:
I thought there was a mypy rule that makes "return as none" implicit if no type is presented?

I could see it either way tho, related thread here: https://github.com/python/mypy/issues/7511

Reply:
Yeah fair enough, Im good to keep as is